debian/oidc-agent.triggers

```diff
@@ -1 +0,0 @@
-activate-noawait update-desktop-database
```

Comment:
This should then be moved to debian/oidc-agent-desktop.triggers

Reply:
Nope; debhelpers 13 have an automatism.

Reply:
using debhelpers 13 in bullseye, buster and focal (20.04). Not on bionic (18.04).
I'll add the triggers back to bionic builds